webapp/components/Editor/ContentViewer.vue

```diff
@@ -39,3 +39,10 @@ export default {
   },
 }
 </script>
+<style lang="scss">
+h3,
+h4,
+hr {
+  margin: 8px 0;
+}
```

Comment:
Would be cool if you could apply these as a class to the HTML tag, see:
https://github.com/Human-Connection/Human-Connection/blob/master/webapp/scss.md

Reply:
Aah! I missed that.

Reply:
This markup is generated by `tiptap`. To add custom class to tag, there should be some mechanism to pass the custom class name via `props`. We will have to dig into TipTap documentation for that. I haven't explored that.

Otherwise, we can increase specificity by using expression 
```
.ProseMirror h3,
.ProseMirror h4,
.ProseMirror hr {
  margin: 24px 0 8px;
}
```
Similar style is followed by `ProseMirror`